code/VersionHistoryExtension.php

```diff
@@ -78,8 +78,19 @@ public function VersionComparisonSummary($versionID = null, $otherVersionID = nu
             if ($versions->count() === 0) {
                 return false;
             }
-            $toRecord = $versions->first();
-            $fromRecord = ($versions->count() === 1) ? null : $versions->last();
+            $toRecord = Versioned::get_version(
+                $this->owner->class,
+                $this->owner->ID,
+                $versions->first()->Version
+            );
+
+            $fromRecord = ($versions->count() === 1)
+                ? null
+                : Versioned::get_version(
+                    $this->owner->class,
+                    $this->owner->ID,
+                    $versions->last()->Version
+                );
```

Comment:
Hi @maxime-rainville this PR looks good overall, I'm just not sure why this one change was necessary. Can you explain please?

Reply:
To be honest, I had completely forgotten about this PR. Off the top of my head, I can't recall why I did this specific change. I think I was getting some sort of error and fetching the version this way was fixing it.